packages/contentstack-migration/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@contentstack/cli-command": "^1.0.1",
     "@contentstack/management": "^1.3.0",
-    "@contentstack/cli-utilities": "^1.0.2",
+    "@contentstack/cli-utilities": "^1.0.3",
```

Comment:
We need to update this package version and all location it is used

Reply:
done